Sources/Overload/OvUI/src/OvUI/Core/UIManager.cpp

```diff
@@ -17,6 +23,8 @@ OvUI::Core::UIManager::UIManager(GLFWwindow* p_glfwWindow, Styling::EStyle p_sty
 	
 	ImGui_ImplGlfw_InitForOpenGL(p_glfwWindow, true);
 	ImGui_ImplOpenGL3_Init(p_glslVersion.data());
+
+	std::filesystem::create_directory(m_layoutsPath);
```

Comment:
Crashes if this gets called when the user starts Overload for the first time (If no `%appdata%/OverloadTech`) folder already exists.
Instead, you can use:
```suggestion
	std::filesystem::create_directories(m_layoutsPath);
```